### PR TITLE
fix(fsdrv): skip the path format if LV_FS_xxx_PATH not defined

### PR DESCRIPTION
**Pull request by xiaoxiang781216**
### Description of the feature or fix

to save the stack space and fix some minor code style issue

### Checkpoints
- [X] Follow the [styling guide](https://github.com/lvgl/lvgl/blob/master/docs/CODING_STYLE.md)
- [ ] Update [CHANGELOG.md](https://github.com/lvgl/lvgl/blob/master/docs/CHANGELOG.md)
- [ ] Update the documentation

### CODE REVIEW

File: src/extra/libs/fsdrv/lv_fs_win32.c [hunk 1]
```diff
@@ -27,7 +24,7 @@
  *  STATIC PROTOTYPES
  **********************/
 
-static bool is_dots_name(LPCSTR name);
+static bool is_dots_name(const char * name);
```
Comment (kisvegabor) [inline src/extra/libs/fsdrv/lv_fs_win32.c:27]: @MouriNaruto please take a look at this.

**Review by kisvegabor — APPROVED** [review]

File: src/extra/libs/fsdrv/lv_fs_win32.c (hunk 1 above)
Comment (MouriNaruto) [reply src/extra/libs/fsdrv/lv_fs_win32.c:27]: @kisvegabor In Windows SDK, `LPCSTR` is `const char*`. So it's OK.

Comment (xiaoxiang781216) [reply src/extra/libs/fsdrv/lv_fs_win32.c:27]: @kisvegabor could you merge PR? @MouriNaruto confirm that the change is good for Windows too.

**Comment by kisvegabor** [comment]
Merged, thanks!

**Pull request merged**